requirements-dev.txt

```
@@ -2,7 +2,7 @@ stestr>=3.0.0
 astroid==2.9.3
 pylint==2.12.2
 black~=22.0
-qiskit-sphinx-theme~=1.12.0
+qiskit-sphinx-theme~=1.13.0
```

Let's you use the ecosystem theme:

```suggestion
qiskit-sphinx-theme~=1.14.0rc1
```